### PR TITLE
Add config.ru and *.rbw to --type ruby

### PR DESCRIPTION
**Pull request by BuonOmo**
- https://stackoverflow.com/questions/55072891/what-is-this-file-config-ru-and-what-is-it-for
- https://fileinfo.com/extension/rbw

### CODE REVIEW

File: crates/ignore/src/default_types.rs [hunk 1]
```diff
@@ -173,7 +173,7 @@ pub const DEFAULT_TYPES: &[(&str, &[&str])] = &[
     ("red", &["*.r", "*.red", "*.reds"]),
     ("robot", &["*.robot"]),
     ("rst", &["*.rst"]),
-    ("ruby", &["Gemfile", "*.gemspec", ".irbrc", "Rakefile", "*.rb"]),
+    ("ruby", &["config.ru", "Gemfile", "*.gemspec", ".irbrc", "Rakefile", "*.rb", "*.rbw"]),
```
Comment (BurntSushi) [inline crates/ignore/src/default_types.rs:176]: Thanks! Could you wrap this to 79 columns (inclusive) like other long lines in this file?

**Review by BurntSushi — CHANGES_REQUESTED** [review]

File: crates/ignore/src/default_types.rs (hunk 1 above)
Comment (BuonOmo) [reply crates/ignore/src/default_types.rs]: @BurntSushi of course :)

Thanks for being so reactive!

**Review by BurntSushi — APPROVED** [review]
Thanks!

**Comment by BuonOmo** [comment]
how soon should I expect a new version ?

**Comment by BurntSushi** [comment]
https://github.com/BurntSushi/ripgrep/blob/master/FAQ.md#release

It does just so happen that I am in the middle of getting out a release now. But I do not put out releases for things like this. Especially when you can add your own file types.

**Pull request merged**